build.zig

```diff
@@ -6,9 +6,9 @@ const example_list = [_][]const u8{
     "streaming-parser",
 };
 
-pub fn build(b: *std.build.Builder) void {
+pub fn build(b: *std.Build) void {
     const gemtext = b.createModule(.{
-        .source_file = .{ .path = "src/gemtext.zig" },
+        .root_source_file = .{ .path = "src/gemtext.zig" },
     });
```

Comment:
I think `addModule("gemtext", …)` is better here, we should also expose the module itself to the public

Reply:
Yes that totally makes sense, thank you!